db/DB.js

```diff
@@ -98,14 +99,31 @@ const DB = {
   },
 
   async _verifySubscriber(emailHash) {
+    // Subscribe user to HIBP
     await HIBP.subscribeHash(emailHash.sha1);
+
+    // Update our subscriber record to verified
     const verifiedSubscriber = await knex("subscribers")
       .where("email", "=", emailHash.email)
       .update({
         verified: true,
         updated_at: knex.fn.now(),
       })
       .returning("*");
+
+    // If the user opted in, send newsletter subscription request to basket
+    if (emailHash.fx_newsletter) {
+      const b = new Basket({BASKET_URL: AppConstants.BASKET_URL, API_KEY: AppConstants.BASKET_API_KEY});
+      // Duplicative calling code; see https://github.com/mozilla/node-basket/issues/4
```

Comment:
nit: Can you add a noisy "TODO:" prefix on that comment, so we can maybe remember to unduplicative that once the upstream magic is merged and published.